src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_bfyx_gemm_like.cpp

```diff
@@ -110,6 +110,12 @@ bool ConvolutionKernel_bfyx_GEMMLike::Validate(const Params& p) const {
         return false;
     }
 
+    // To prevent big sized filter which causes lots of CL build time.
+    const size_t acceptalbe_filter_x_size = 64;     // This acceptable size was decided by heuristics
```

Comment:
why the condition does not include y-axis? why is it different from the condition from `convolution_kernel_bfyx_os_iyx_osv16.cpp`?

Reply:
convolution_kernel_bfyx_gemm_like kernel creates LOOP jit constants by filter's x-axis value.